packages/shared/src/graphql/fragments.ts

```diff
@@ -117,15 +117,11 @@ export const COMMENT_FRAGMENT = gql`
   fragment CommentFragment on Comment {
     id
     contentHtml
-    content
```

Comment:
Just double check we really do not need content, this fragment is used all over the place 🙏

Reply:
Yup, I only added §content§ field in the previous PR and this was the only place it was used AFAIK.